TWLight/resources/templates/resources/partner_detail.html

```diff
@@ -175,14 +175,12 @@ <h1>{{ object }}</h1>
               We filter out details and summary tags so that we always see
               the full content on the details page.
             {% endcomment %}
-            {% filter twlight_removetags:"details summary" %}
-              {% cache 31536000 partner_description LANGUAGE_CODE object.pk %}
-                {{ object.description | twlight_wikicode2html | safe }}
-              {% endcache %}
-            {% endfilter %}
+            {% cache 31536000 partner_short_description LANGUAGE_CODE object.pk %}
+              {{ object.short_description | twlight_wikicode2html | safe }}
+            {% endcache %}
             </p>
           {% else %}
-            {% comment %} Translators: If a partner has no description written, this message is shown in the Description field on the partner page. {% endcomment %}
+            {% comment %} Translators: If a partner has no short_description written, this message is shown in the Description field on the partner page. {% endcomment %}
```

Comment:
short_description is an internal variable name and doesn't need to be shown to translators. 'description' is fine here.